testsuite/MDAnalysisTests/test_transformations.py

```diff
@@ -297,8 +297,11 @@ def test_shear_from_matrix():
     # direct = np.random.random(3) - 0.5
     # point = np.random.random(3) - 0.5
     # normal = np.cross(direct, np.random.random(3))
+    # In this random configuration the test will fail about 0.05% of all times.
+    # Then we hit some edge-cases of the algorithm. The edge cases for these
+    # values are slightly different for the linalg library used (MKL/LAPACK).
     # So here are some of my random numbers
-    angle = 2.8965075413405783
+    angle = 2.8969075413405783
```

Comment:
Perhaps instead we should just define this to 3 dp as it seems that that's all we're confident up to.  The extra digits imply certainty (my mistake, I know)